app/code/community/Yotpo/Yotpo/Helper/Data.php

```diff
@@ -54,9 +54,9 @@ private function renderYotpoProductBlock($thisObj, $blockName, $product = null,
         if ($product != null) {
             $block->setAttribute('product', $product);
         }
-        if ($block != null) {
+		if ($block != null) {
             if ($print == true) {
-                $block->toHtml();
+               return $block->toHtml();
```

Comment:
@ijazalise can we just leave it as it was? i see here additional return that you've added. unless I am missing something and this code is relevant to your changes

Reply:
@ijazalise ?